### ISSUE TITLE
Related publications on the gene pages (?)

### ISSUE THREAD

**Issue by amccoy95** [mpi2/PhenotypeData#223]
Suggestion came directly from new IMPC comms officer Emma. 

When going through the IMPC website she pointed out there's no publications mentioned on the gene pages. Is this something that has been dicussed before?

**Comment by viomunoz** [comment]
Yes, we have. This is something we could do, also include links to secondary phenotyping projects on genes produced by the IMPC. Currently, you can go to the publications section and filter for your gene of interest.

**Comment by amccoy95** [comment]
I think having these sections would be really beneficial.

We'll also have the advantage of linking to the publications area of the website from the gene page. This means general browsing can extend to more than just the data but also our publications. 

I imagine this isn't high priority for the time being but would be good to get the ball rolling on this.

Would everyone be happy for me to bring this up in the Comms Call today?

**Comment by viomunoz** [comment]
We have discussed it in the Comms Call before, I brought it up in two occasions. Not much feedback so far.

**Comment by viomunoz** [comment]
Some points to consider

- Do we want to include only curated papers, i.e. those in https://www.mousephenotype.org/data/publications or do we favour performing a literature search for papers mentioning this gene? In either case, we need to clearly state which option we are following.

- We need to think how to cite Consortium papers. Do we want to include them, for example, in a separate top (?) section or highlighting them somehow? Looking for the genes that were included in each consortium paper will require a bit more work, but hopefully the DR version should be stated in all of them.

**Comment by elambourne** [comment]
A quick update since lasts week's F2F for record's sake:
This was brought up during the breakoff group. Terry's advice was to potentially not go ahead with this idea as it could clutter the page. 
Are we still planning on testing this out of dev?

**Comment by amccoy95** [comment]
Cluttering the page is a good concern. 

However we do have these floating headers above each section, which to me screams accordion. We can make each heading a clickable tab which can show and hide the data below it. This state could be saved for returning users who know what data isn't relevant for thier needs.

So publication will be added to the bottom of the page, below the Order section. Maybe only the publications section can be collapsible instead of the other sections? 

I think Terry's concern was the overview section at the top and the addition of a publications icon; but I don't think that needs to be added.

![image](https://user-images.githubusercontent.com/46707334/76303110-1809f900-62b9-11ea-9fd2-1c77d992232d.png)

**Comment by viomunoz** [comment]
Other issues identified included: (1) how to select what publications to add; and (2) anybody looking for publications on a gene of interest will most likely use specialized bibliographic search engines.
